packages/blob/src/index.ts

```diff
@@ -1,7 +1,9 @@
+type BlobPart = BufferSource | Blob | string;
```

Comment:
`BlobPart` should be a native built-in type, after all, it's used in the `Blob` constructor. You don't need to define it here.